### PR TITLE
Upgrade to 1.17.1

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to 1.16.9

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
@Josue-T : can you elaborate what's the regression about ? what doesn't work exactly ?

**Comment by Josue-T** [comment]
Before this version the SSO worked well and now It's broken by some change on gitea side and I didn't found the solution to fix the SSO as it should work. cf: https://github.com/go-gitea/gitea/issues/16798

File: conf/app.ini [hunk 1]
```diff
@@ -40,7 +40,7 @@ ENABLE_NOTIFY_MAIL     = true
 DISABLE_REGISTRATION   = true
 ENABLE_CAPTCHA         = false
 REQUIRE_SIGNIN_VIEW    = false
-ENABLE_REVERSE_PROXY_AUTHENTICATION = false
+ENABLE_REVERSE_PROXY_AUTHENTICATION     = true
 ENABLE_REVERSE_PROXY_AUTO_REGISTERATION = false
```
Comment (tituspijean) [inline conf/app.ini:44]:
```suggestion
ENABLE_REVERSE_PROXY_AUTO_REGISTRATION = false
```

**Comment by ericgaspar** [comment]
!testme

**Comment by Josue-T** [comment]
!testme

**Comment by alexAubin** [comment]
eeeeh, !testme ?!

**Comment by ericgaspar** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
@Josue-T, seems ready

**Comment by yalh76** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 2]
```diff
@@ -74,21 +73,26 @@ if [[ $migration_process -eq 1 ]]; then
...
             ynh_secure_remove --file=$p
         done
     fi
 
     upstream_version="0.0.1"
 fi
 
+# Move data directory
+if [ -e "/home/""$app" ] && [ ! -e $datadir ]; then
+    mv "/home/""$app" "$datadir"
```
Comment (ericgaspar) [inline scripts/upgrade:93]: @Josue-T If I am not mistaken we are moving  `"/home/$app"` to `"/home/$app"`

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by Josue-T** [comment]
!testme

**Comment by Josue-T** [comment]
!testme

**Comment by ndarilek** [comment]
Is the SSO issue still a thing? If so, what is it?

I've been using this branch successfully since February or so. There *may* have been an initial SSO issue, but so far I've been able to sign on fine.

Concerned because 1.14.5 is ancient and probably has security implications.

Thanks.

**Review by tituspijean — APPROVED** [review]
Upgrade tested on my VPS, running fine and no issue with authentication.

**Review by Tagadda — APPROVED** [review]
:rocket:

**Comment by ericgaspar** [comment]
I hope this branch will merge before Gitea gets federation support 😅

**Pull request merged**